data/functions.sql

```diff
@@ -1006,28 +1006,31 @@ BEGIN
     RETURN '{}'::jsonb;
   END IF;
 
-  -- first, try the countries table
-  SELECT
-    min_label, max_label INTO min_zoom, max_zoom
-    FROM ne_10m_admin_0_countries c
-    WHERE c.wikidataid = wikidata_id;
+  -- if it's a country, only look it up in the iso and tlc tables
+  IF place_tag='country' OR place_tag='unrecognized' THEN
+      SELECT
+        min_label, max_label INTO min_zoom, max_zoom
+        FROM ne_10m_admin_0_countries_iso i
+        WHERE i.wikidataid = wikidata_id;
```

Comment:
This will get you Taiwan, even though it's not `IN ('Admin-0 country' or 'Admin-0 dependency');`, nice

Reply:
It'll also yield the French overseas regions (mostly fine).